migrations/248_instance_questions__requires_manual_grading__add.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE instance_questions ADD COLUMN IF NOT EXISTS requires_manual_grading BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE instance_questions ADD COLUMN IF NOT EXISTS assigned_grader BIGINT;
+ALTER TABLE instance_questions ADD COLUMN IF NOT EXISTS last_grader BIGINT;
+ALTER TABLE instance_questions ADD FOREIGN KEY (assigned_grader) REFERENCES users ON DELETE SET NULL ON UPDATE CASCADE;
+ALTER TABLE instance_questions ADD FOREIGN KEY (last_grader) REFERENCES users ON DELETE SET NULL ON UPDATE CASCADE;
+
+WITH ranked_submissions AS (
+    SELECT
+        v.instance_question_id,
+        s.*,
+        ROW_NUMBER() OVER (PARTITION BY v.instance_question_id ORDER BY s.date DESC) AS submission_row
+    FROM
+        variants AS v
+        JOIN submissions AS s ON (s.variant_id = v.id)
+)
+UPDATE instance_questions AS iq
+SET
+    requires_manual_grading = TRUE
+FROM
+    assessment_questions AS aq
+    JOIN questions AS q ON (q.id = aq.question_id)
+    JOIN ranked_submissions AS rs ON (TRUE)
+WHERE
+    aq.id = iq.assessment_question_id
+    AND q.grading_method = 'Manual'
+    AND rs.instance_question_id = iq.id
+    AND rs.submission_row = 1
+    AND rs.graded_at IS NULL;
```

Comment:
If this PR were to be deployed in production with migrations being done separately from code changes, this update should probably be done just before code changes are implemented to avoid discrepancies. Not sure what the best approach should be in that case.

Reply:
@mwest1066 @nwalters512 any thoughts on this?